File: matchmaker/server/src/main/java/ru/ifmo/se/dating/matchmaker/api/ResponseEntityStub.java
package ru.ifmo.se.dating.matchmaker.api;

import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;

final class ResponseEntityStub {
    private ResponseEntityStub() {

    }

    static <T> ResponseEntity<T> create() {
        return ResponseEntity.status(HttpStatus.I_AM_A_TEAPOT).build();
    }
}

File: people/server/src/main/kotlin/ru/ifmo/se/dating/people/api/ResponseEntityStub.kt
package ru.ifmo.se.dating.people.api

import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity

internal object ResponseEntityStub {
    fun <T> create(): ResponseEntity<T> =
        ResponseEntity.status(HttpStatus.I_AM_A_TEAPOT).build()
}
